QUIZ APP/quiz.js

```diff
@@ -0,0 +1,228 @@
+                { text: "Using the cheapest available raw materials", correct: false },
+        ],
+        explanation: "ANSWER: Maintaining personal hygiene in the workplace is the correct answer",
+      },
+    
+      { 
+        id: 3,
+        type: "GMP",
+        question: "What is the primary goal of Good Manufacturing Practice (GMP) regulations?",
+        answers: [
+                { text: "To maximize production output", correct: false },
+                { text: "To ensure products are safe and of high quality", correct: true },
+                { text: "To minimize the cost of production", correct: false },
+                { text: "To increase market share", correct: false },
+        ],
+        explanation: "ANSWER: To ensure products are safe and of high quality is the correct answer",
+      },
+      { 
+        id: 4,
+        type: "Personal Hygiene",
+        question: "Knowing the correct way to handle the products hygienically is the responsibility of:",
+        answers: [
+                { text: "Only from health authorities", correct: false },
+                { text: "All the personal staff of the company", correct: true },
+                { text: "Only from consumers", correct: false },
+                { text: "Only the owners of the company", correct: false },
+        ],
+        explanation: "ANSWER: All the personal staff of the company is the correct answer",
+      },
+      { 
+        id: 5,
+        type: "GMP",
+        question: "Why GMP are so important in the company",
+        answers: [
+                { text: "To increase company profits", correct: false },
+                { text: "Reduce risks and avoid contamination", correct: true },
+                { text: "To minimize the cost of production", correct: false },
+                { text: "Improve product quality", correct: true },
+        ],
+        explanation: "ANSWER: Reduce risks and avoid contamination and Improve product quality are the correct answer",
+      },
+    ];
+
+    const playerName = localStorage.getItem("playerName");
+    const playerNameElement = document.getElementById("playerName");
+    const quizSection = document.getElementById("quizSection");
+
+    playerNameElement.textContent = playerName; // Muestra el nombre del jugador en la página
+
+    const questionElement = document.getElementById("question");
+    const answerButtons = document.getElementById("answerButtons");
+    const nextButton = document.getElementById("btn-next");
+    const shuffleButton = document.getElementById("shuffleButton");
+    const explanationElement = document.getElementById("explanation");
+    
+    playerNameElement.textContent = playerName;
+    
+    let currentQuestionIndex = 0;
+    let score = [0];
+    let player = "";
+    let currentPlayerIndex = 0;
+    
+    
+    
+    
+    
+    
+    // startButton.addEventListener("click", startButtonClick);
```

Comment:
It's better not to commit commented code to the repository. If you ever need to go back, you can use the git history.